Documentation/GeneralConventions/ReviewPolicy.rst

```diff
@@ -0,0 +1,201 @@
+    our maintainers will take care of it - this is just a step to make
+    their work easier.
+*   Please provide a **meaningful title** to your Pull Request.
+*   Please provide a **short description** of what your Pull Request
+    is about.
+*   Please try to **focus your Pull Request on a single issue/topic**. At best,
+    a pull request should only affect a single file or chapter, maybe even
+    a section. Feel free to contribute multiple Pull Requests for multiple
+    topics. This makes our review process quicker and easier.
+*   **Be open to feedback** that our maintainers may have about the change,
+    and sometimes there will be questions or other comments that will help
+    to finalize your contribution.
+*   Please **be patient for feedback** to your contribution. Our maintainers
+    work on Pull Requests in their spare time.
+
+If you are interested in the work of our maintainers, read on in the
+next section.
+
+..  _review-policy-maintainers:
+
+Review policy for maintainers
+-----------------------------
+
+This section is intended for current maintainers, but please read on,
+if you are interested in maybe becoming one in the future.
+
+**Contributions are very valuable**: Anyone putting in the effort and time
+to help us should **feel rewarded** by doing so. We try to adjust our review
+policy to making this possible without sacrificing the **quality of our
+documentation**.
+
+The review process should have the **same standard** for anyone of the
+maintainer team performing the review, and every member of the team
+should feel good when **making decisions** on what and how to give feedback on.
+
+Basic work on contributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*   **Be kind and understanding**. Even when rejecting a contribution,
+    there is a person behind a text, who meant well.
+*   Try to **improve and explain changes** made to Pull Requests.
+*   The automatic **Code-Checks** must be green (maintainers may need to
+    execute them first for first-time contributors - before you do, ensure
+    no malicious code is contained in `.github` action code changes or general
+    code snippets)
+*   Ensure **factual correctness** and **basic spell-checks** of a Pull Request.
+*   A maintainer can and should always **directly edit/commit a Pull Request** to
+    apply the following changes without the need to first ask for
+    feedback of the contributor or other team members:
+
+    *   **Apply spelling fixes**.
+    *   **Fix syntax errors** made in code.
+    *   **Fix mistakes in reST markup** (indentation, wrong directives, missing headers,
+        wrong or missing linebreaks, wrong or bad links/references, ...) - see
+        :ref:`format-rest-cgl`.
+
+Workflow and follow-ups to contributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following more specific checks to be fulfilled, before a Pull Request can be
+merged:
+
+*   Ensure the Pull Request is **made against the most recent applicable
```

Comment:
I would use linkable headlines for each block point in the list. so that I can say: "mergers can be oppininated, see here: "

We can also do that in a follow up.

Reply:
Sprinkled in a few of those, like this?

Reply:
Let us deal with such details in a follow up and get this merged first.